pages.es/linux/distrobox.md

```diff
@@ -0,0 +1,37 @@
+# distrobox
+
+> Utiliza cualquier distribución Linux dentro de su terminal en un contenedor. Instala y utiliza paquetes dentro de ella mientras se integra estrechamente con el sistema operativo anfitrión, compartiendo el almacenamiento (directorio `home`) y el hardware.
```

Comment:
> Utiliza cualquier distribución Linux dentro de su terminal en un contenedor. 
> Instala y utiliza paquetes dentro de ella mientras se integra estrechamente con el sistema operativo anfitrión, compartiendo el almacenamiento (directorio `home`) y el hardware.

Reply:
Separated paragraphs and added a `>` in the paragraph that begins with` Instala y utiliza (...)`